storage/ndb/rest-server/data-access-rondb/src/db-operations/pk/common.cpp

```diff
@@ -102,11 +97,10 @@ RS_Status SetOperationPKCol(const NdbDictionary::Column *col, NdbOperation *oper
     if (unlikely(*parsed != '\0' || errno != 0 || !(parsed_number >= 0 && parsed_number <= 255))) {
       return RS_CLIENT_ERROR(ERROR_015 + std::string(" Expecting TINYINT UNSIGNED. Column: ") +
                              std::string(request->PKName(colIdx)));
-    } else if (unlikely(operation->equal(request->PKName(colIdx),
-                                         static_cast<Uint8>(parsed_number)) != 0)) {
-      return RS_SERVER_ERROR(ERROR_023);
     }
-    return RS_OK;
+    primary_key      = &parsed_number;
```

Comment:
In the previous patch you spent a lot of time on changing to CamelCase, now you introduce new variables not using CamelCase, not sure I follow what the standard is here.

Reply:
Yes, this is wrong, changed it to camel case